docs/JSON-RPC.md

```diff
@@ -186,6 +186,22 @@ Results:
 | result.clients | array | The client list. See jamulusclient/clientListReceived for the format. |
 
 
+### jamulusclient/pollServerList
+
+Requests server list.
+
+Parameters:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| params.directory | string | URL of directory, e.g. anygenre1.jamulus.io:22124 |
```

Comment:
URL maybe call it host and port. Not really what I'd use understand d as URL

Reply:
Okay, it does not include the protocol prefix. I will change this.

Reply:
I have used "address" for the parameter (which is what I had originally used but changed it!) and "socket address" for descriptions. According to [Wikipedia](https://en.m.wikipedia.org/wiki/Network_socket) a combination of IP address and port number is known as a socket address.

Reply:
Thanks!